src/base/ddl_log.c

```diff
@@ -658,7 +658,7 @@ logddl_make_copy_dir (T_APP_NAME app_name, char *copy_filename, char *copy_fullp
   const char *env_root = NULL;
   int retval = 0;
 
-  if (copy_filename == NULL || copy_fullpath == NULL || buf_size < 0)
+  if (copy_filename == NULL || copy_fullpath == NULL)
```

Comment:
I think it would be better to remove the buf_size argument and specify PATH_MAX directly.

Reply:
I think the buf_size parameter should be kept. The reason is that the size of the string can change.